### DISCUSSION TITLE
Need to Allow Remote Location

### DISCUSSION THREAD

**Discussion by larry72450** [SafeExamBrowser/seb-win-refactoring#579] [General]
A little background.  We use windows 2016 Standard and running vspace from Ncomputing for years.  I setup Safe Exam Browser and it has been workin great taking test on Moodle.  It does use remote desktop licenses.  And every user has a thin client. We use L300 and L350's

Since I formattted the hard drive on the server and using Windows 2016 Standard, I cannot get Safe Exam Browser to complete the startup.  I got it installed.

I am using the same config file.  When I tried to use it, it gives a popup the "The System appears to be running in a remote session.  The selected configuration does nto allow SEB to be run in a remote session.

I need to allow users from the thin clients to use Safe Exam Browser.  The old SEB version worked, but not the 3.4.1

What do I need to change, or what version will allow this to work?

Thanks,

**Comment by dbuechel** [comment]
You'd then need to enable the setting "Allow remote session/screen sharing", see https://safeexambrowser.org/windows/win_usermanual_en.html#SecurityPane.
